jfrog-cli/utils/tests/utils.go

```diff
@@ -308,3 +309,11 @@ func GetBaseDir(setToParentDir bool) (baseDir string) {
 	baseDir = filepath.Join(pwd, "testdata")
 	return
 }
+func DeleteUtilForCleanUp(deleteSpec *spec.SpecFiles, flags *generic.DeleteConfiguration) (successCount, failCount int, err error) {
```

Comment:
I suggest renaming this function to *DeleteFiles*.
There's a missing new-line before this function.